features/unsupported/fs/sd/SDFileSystem.h

```diff
@@ -58,6 +58,7 @@ class SDFileSystem : public FATFileSystem {
     virtual int disk_write(const uint8_t* buffer, uint32_t block_number, uint32_t count);
     virtual int disk_sync();
     virtual uint32_t disk_sectors();
+	void debug(bool dbg);
```

Comment:
using tabs here? some lines in this file and the other are misaligned here on github